pkgs/tools/networking/slack-cli/default.nix

```diff
@@ -0,0 +1,27 @@
+# slack-cli must be configured using the SLACK_CLI_TOKEN environment
+# variable. Using `slack init` will not work because it tries to write
+# to the Nix store.
```

Comment:
It could be worked around by changing the location it saves to, but then a decision would have to be made where that should be. A wrapper script that warns if the environment variable is not set is a good idea though.

Reply:
Sounds like that should be in the home directory. Where does it regularly save it?

But either way, your call. I don't use the application. Just silently failing without giving the user a hint for the reason is sub-optimal.

Reply:
It looks like it wants to save it to `etc`, which seems very odd to me, because then it can't be used by multiple users. I think that changing it only for nixpkgs would be confusing, so I'm going to leave it as-is, since I'm happy using the environment variable. That way if somebody else wants to use the file approach they can change it to be what they think makes sense.

Reply:
Yes, that seems reasonable (unlike the upstream approach of saving user configuration to `/etc`). But then please add a wrapper that notifies the user and maybe add a comment explaining the alternative solution (in case somebody wants to improve it in the future).

Reply:
Will do.